dovpanda/core.py

```diff
@@ -94,9 +94,9 @@ def csv_index(res, arguments):
 @ledger.add_hint('read_csv', 'pre')
 def check_csv_size(arguments):
     filename = arguments.get('filepath_or_buffer')
-    if getsize(filename) > MAX_CSV_SIZE:
+    if getsize(filename) > config.MAX_CSV_SIZE:
         ledger.tell('File size is very large. To avoid format issues before the complete file loades, '
```

Comment:
"format issues" is unclear.
go with:
file size is very large and may take time...
if you would like to avoid... try {code} to check schema...